testutils/hadoop

```diff
@@ -69,6 +69,7 @@ fi
 HADOOP_OPTS="$HADOOP_OPTS -Dhadoop.policy.file=$HADOOP_POLICYFILE"
 
 # Disable ipv6 as it can cause issues
+# TODO: change to -Djava.net.preferIPv4Stack=false after Hadoop adds support for IPv6
 HADOOP_OPTS="$HADOOP_OPTS -Djava.net.preferIPv4Stack=true"
```

Comment:
Can we assume libraries provided by Hadoop can handle IPv6?

Reply:
No, Hadoop presently doesn't support IPv6

Reply:
I guess this PR assumes the environment where Hive is deployed without YARN or HDFS. Just to clarify, I still wonder if shared or client libraries used in Hive, such as hadoop-common or File System API, correctly work with IPv6. Is the answer probably yes?

Reply:
You are right, this PR has to wait for Hadoop to add IPv6 support.
Hadoop currently doesn't support IPv6, and I haven't found info about any hadoop libraries to support IPv6.

Reply:
I understand your point.

Reply:
I checked the documents of Java 8 and 17 just in case.
- https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html
- https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/net/doc-files/net-properties.html